hs_composite_resource/models.py

```diff
@@ -1,6 +1,7 @@
 import os
 
 from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
 
```

Comment:
Are there considerations for different environments? Dev/UAT/QA/beta/prod? If so, make sure deployment staff understand the implications to Jenkins, settings, local_settings